winpr/libwinpr/utils/cmdline.c

```diff
@@ -322,6 +322,16 @@ int CommandLineParseArgumentsA(int argc, LPSTR* argv, COMMAND_LINE_ARGUMENT_A* o
 
 						options[j].Flags |= COMMAND_LINE_VALUE_PRESENT;
 					}
+					else
+					{
+						/*
+						We need to reset the option Value and Flag
+						to process correctly the cases where the same
+						option is given several times like: /foo:bar /foo
+						*/
+						options[j].Value = NULL;
```

Comment:
looks like a misuse of the command line, usually it should be `/option:foo,bar,bla`